### DISCUSSION TITLE
Error with installation of components on npm only systems

### DISCUSSION THREAD

**Discussion by VyoJ** [shadcn-ui/ui#5418] [Q&A]
On trying to add components which use radix-ui, I get the following error:

```
✔ Checking registry.
⠙ Installing dependencies.
Something went wrong. Please check the error below for more details.
If the problem persists, please open an issue on GitHub.

Command failed with exit code 1: pnpm add @radix-ui/react-avatar
'pnpm' is not recognized as an internal or external command,
operable program or batch file.
```

I am using a system which has only npm installed on it, however I have not faced this issue in previous projects built with shadcn/ui. 

I can manually install the dependencies and add the component code. However, is it necessary to only use pnpm now with the new CLI for adding components which have external dependencies?

Can be reproduced on systems without npm with: `npx shadcn@latest add <components_with_external_dependencies>`
Example components can be: Command, Avatar, Popover, etc. (anything which has a radix-ui dependency I suppose)

**Comment by eduardovargasleffa** [comment]
Remove root file `.pnpm`. This work for me.
